### DISCUSSION TITLE
Meta Thread for MacOS issues

### DISCUSSION THREAD

**Discussion by majestrate** [oxen-io/lokinet#2094] [General]
please report any issues you encounter with macos in this thread. please include lokinet version number and macos version number. 

we will spin off new issues on the github issue tracker for each of them to track their progress.

**Comment by sbecker11** [comment]
version: lokinet-0.9.10-2c69d6bed
Lokinet address: 67zwimkb9ye888fn3fpdfrrcx5fdspw6rz1a9j5msge39q4b6nuy.loki
MacOS x86_64 Ventura 13.2.1 

Labels on the vertical axis of the chart are truncated improperly.:

![Screenshot 2023-03-27 at 4 41 15 PM](https://user-images.githubusercontent.com/4529120/228083076-19843742-95da-494c-8b34-d290748b2ed6.png)

**Comment by sbecker11** [comment]
version: lokinet-0.9.10-2c69d6bed
Lokinet address: 67zwimkb9ye888fn3fpdfrrcx5fdspw6rz1a9j5msge39q4b6nuy.loki
MacOS x86_64 Ventura 13.2.1 

There are no visible means of submitting support requests

**Comment by sbecker11** [comment]
version: lokinet-0.9.10-2c69d6bed
Lokinet address: 67zwimkb9ye888fn3fpdfrrcx5fdspw6rz1a9j5msge39q4b6nuy.loki
MacOS x86_64 Ventura 13.2.1 

click on the lokinet icon in the top-right menu bar to open the lokinet modal
click on the top-left moon icon of the modal to toggle the display to night mode
from the top-left menu bar for the Lokinet app, choose View / Toggle Full Screen
click the X link at the top-right of the modal to close the modal
the entire screen goes black 
the top menu buttons are hidden
no visible means are provided to get out of full-screen mode
